awot/util/write_kmz.py

```diff
@@ -23,6 +23,7 @@
     import simplekml
 except:
     raise ValueError("This module requires installation of simplekml...")
+#    return
```

Comment:
I don't think this return needs to be here as it's an import test. Commented out, but maybe we should remove to clear up any confusion later.

Reply:
Have you found these kmz writers to be useful?